Makefile

```diff
@@ -4,7 +4,7 @@ BUNDLE_VERSION = 4.7.4
 BUNDLE_EXTENSION = crcbundle
 CRC_VERSION = 1.24.0
 COMMIT_SHA=$(shell git rev-parse --short HEAD)
-MACOS_INSTALL_PATH = /Library/CodeReady Containers
+MACOS_INSTALL_PATH = /Applications/CodeReady Containers.app/Contents/Resources/
```

Comment:
This probably deserves some followup to have 
```
MACOS_INSTALL_PATH = /Applications/CodeReady Containers.app
MACOS_RESOURCES_PATH = /Applications/CodeReady Containers.app/Contents/Resources/`
```
this would make sense at least in the go codebase.